app/experimenter/experiments/models/nimbus.py

```diff
@@ -0,0 +1,99 @@
+from django.contrib.auth import get_user_model
+from django.contrib.postgres.fields import ArrayField
+from django.core.validators import MaxValueValidator
+from django.db import models
+from django.utils import timezone
+
+from experimenter.projects.models import Project
+from experimenter.experiments.constants import ExperimentConstants
+
+
+class NimbusExperiment(ExperimentConstants, models.Model):
+    owner = models.ForeignKey(
+        get_user_model(),
+        blank=True,
+        null=True,
+        on_delete=models.CASCADE,
+        related_name="owned_nimbusexperiments",
+    )
+    related_to = models.ManyToManyField(
+        "experiments.NimbusExperiment", blank=True, related_name="related_by"
+    )
+    parent = models.ForeignKey(
+        "experiments.NimbusExperiment", models.SET_NULL, blank=True, null=True
+    )
+    status = models.CharField(
+        max_length=255,
+        default=ExperimentConstants.STATUS_DRAFT,
+        choices=ExperimentConstants.STATUS_CHOICES,
+    )
+    archived = models.BooleanField(default=False)
+    name = models.CharField(max_length=255, unique=True, blank=False, null=False)
+    slug = models.SlugField(max_length=255, unique=True, blank=False, null=False)
+    public_description = models.TextField(blank=True, null=True)
+    short_description = models.TextField(default="", blank=True, null=True)
+    is_paused = models.BooleanField(default=False)
+    proposed_start_date = models.DateField(blank=True, null=True)
+    proposed_duration = models.PositiveIntegerField(
+        blank=True,
+        null=True,
+        validators=[MaxValueValidator(ExperimentConstants.MAX_DURATION)],
+    )
+    proposed_enrollment = models.PositiveIntegerField(
+        blank=True,
+        null=True,
+        validators=[MaxValueValidator(ExperimentConstants.MAX_DURATION)],
+    )
+    firefox_min_version = models.CharField(
+        max_length=255,
+        choices=ExperimentConstants.VERSION_CHOICES,
+        blank=True,
+        null=True,
+    )
+    firefox_max_version = models.CharField(
+        max_length=255,
+        choices=ExperimentConstants.VERSION_CHOICES,
+        blank=True,
+        null=True,
+    )
+    firefox_channel = models.CharField(
+        max_length=255,
+        choices=ExperimentConstants.CHANNEL_CHOICES,
+        blank=True,
+        null=True,
+    )
+    projects = models.ManyToManyField(Project, blank=True)
+    objectives = models.TextField(
+        default=ExperimentConstants.OBJECTIVES_DEFAULT, blank=True, null=True
+    )
+    bugzilla_id = models.CharField(max_length=255, blank=True, null=True)
+    recipe_slug = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Don't need this, see https://mozilla.slack.com/archives/C011MC5EB0V/p1599767736076000